.travis.yml

```diff
@@ -30,8 +30,9 @@ install:
   - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && ./gradlew :ui:assemble --stacktrace || ./gradlew --stacktrace '
 
 script:
+  - if [[ "TRAVIS_OS_NAME" == "osx" ]]; then ./gradlew :ui:test -Pgeneration; else
```

Comment:
So what is with the spaces followed by a - followed by another space?
Also that will eventually be removed once I get the generation tests running on Linux as well.